packages/react-core/src/components/NotificationDrawer/NotificationDrawerListItem.tsx

```diff
@@ -54,6 +65,7 @@ export const NotificationDrawerListItem: React.FunctionComponent<NotificationDra
       onClick={(e) => onClick(e)}
       onKeyDown={onKeyDown}
     >
+      <span className="pf-v5-screen-reader">{readStateSRText}</span>
```

Comment:
We'll most likely need an `aria-live="polite"` added somewhere, as right now there's no indication that anything has changed when marking a notification as read. Personally I lean a bit more towards adding it to the `pf-v5-c-notification-drawer__header-status` element (line ~45 of NotificationDrawerHeader), so that the current count is announced when updated. Doing that would:

- provide users the context of the read status of the current notification item when traversing through the list from this `readStateSRText`
- alert users that the notification they are on is marked as read by the announcement of "[number] unread" from the notification drawer header.

Reply:
@jessiehuff I would be interested in your opinion here as well.  I know we talked about a couple of different options.

Reply:
When testing on VO, I noticed that when I "read" a notification, the screen reader does currently say that specific notification is read. (It looks like from your comment that that was true for you as well. Though it might be wise to check this in other screen readers like JAWS.) If I'm understanding correctly, we're thinking of making the "2 unread" notifications the live region and then updating that text every time a user reads a notification? I suppose that as long as the live region is set to polite, that wouldn't be too bad because then it ideally should let the user get through the notification they're on before informing them of what they have left. I think that could make sense to me.